translate_model/postprocess.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
so, the absolute parent would be model/translate/postprocess?

how i am visualizing it is 
1. models directory -> will have translate and lasertagger as 2 sub-folders
2. dataset directory -> will have wikisplit, translate and rule-based as 3 sub-folders
3. utils directory -> can have translate and data as 2 sub folders

Please add if i missed something

Reply:
the utils directory is currently within the models directory. Should i move it out?

Reply:
is it being used within dataset directory?
i am supposing yes since translate dataset generation might be using it. if yes, then yes lets move it out of models but if the utls are being used only by models, let it put

Reply:
yes, that makes sense. I'll move it out of the models directory.